uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -30,7 +30,6 @@ define([
         mode = '', //can be edit, create, view
         role = '', //could be manager, owner or admin
         selectedAbsenceType = {},
-        serverDateFormat = 'YYYY-MM-DD',
```

Comment:
why not assign here itself instead of calling it at several places in code ?
`serverDateFormat = sharedSettings.serverDateFormat`